test/remote/gateways/remote_redsys_rest_test.rb

```diff
@@ -0,0 +1,425 @@
+require 'test_helper'
+
+class RemoteRedsysRestTest < Test::Unit::TestCase
+  def setup
+    @gateway = RedsysRestGateway.new(fixtures(:redsys))
+    @gateway_3ds = RedsysRestGateway.new(fixtures(:redsys_3ds))
+    @amount = 100
+    @credit_card = credit_card('4548812049400004')
```

Comment:
I didn't find this test card number on their docs, is the a valid non-3DS card?

Reply:
It was a 3DSv1 test card a while ago, but 3DSv1 was sunset in 2022.